frontends/election-manager/src/app.tsx

```diff
@@ -1,38 +1,50 @@
+import { Logger, LogSource } from '@votingworks/logging';
 import { Provider } from '@votingworks/types';
 import {
   getHardware,
   getPrinter,
-  KioskStorage,
-  LocalStorage,
+  Storage,
   WebServiceCard,
 } from '@votingworks/utils';
-import React from 'react';
+import React, { useMemo } from 'react';
 import { BrowserRouter } from 'react-router-dom';
 import './App.css';
 import { AppRoot, Props as AppRootProps } from './app_root';
 import { getConverterClientType } from './config/features';
 import { MachineConfig } from './config/types';
+import { ElectionManagerStoreStorageBackend } from './lib/backends';
 import { machineConfigProvider } from './utils/machine_config';
 
 export interface Props extends Partial<AppRootProps> {
   machineConfig?: Provider<MachineConfig>;
+  storage: Storage;
 }
 
-const defaultStorage = window.kiosk
-  ? new KioskStorage(window.kiosk)
-  : new LocalStorage();
-
 export function App({
+  backend: backendFromProps,
   hardware = getHardware(),
   card = new WebServiceCard(),
-  storage = defaultStorage,
+  storage,
```

Comment:
does `App` need both `backend` and `storage`? Or can it just take `backend`?

Reply:
Yes, you're right. This just expanded the scope of the change but I think you're right that it doesn't make sense to leave it in this intermediate state.

Reply:
Done.